src/patternfly/components/Dropdown/dropdown-toggle.hbs

```diff
@@ -1,18 +1,19 @@
-<button class="pf-c-dropdown__toggle{{#if dropdown-toggle--modifier}} {{dropdown-toggle--modifier}}{{/if}}"
-  id="{{id}}" 
-  aria-haspopup="true" 
-  aria-expanded="{{#if dropdown--IsExpanded}}true{{else}}false{{/if}}"
-	{{#if aria-label}}
-		aria-label="{{aria-label}}" 
-	{{/if}}
-	{{#if dropdown-toggle--attribute}}
-		{{{dropdown-toggle--attribute}}}
-	{{/if}}>
-	{{> @partial-block}}
-	{{#if dropdown--HasToggleIcon}}
-		{{> dropdown-toggle-icon}}
-	{{/if}}
-	{{#if dropdown--HasKebabIcon}}
-		{{> kebab-icon}}
-	{{/if}}
-</button>
+<button id="{{id}}-button"
```

Comment:
no big deal but this file has some indentation issues.

Reply:
😢

Reply:
I wonder if there's an issue in your editor settings where it's inserting more spaces than you see visually? Check out https://github.com/patternfly/patternfly-next/blob/1c64c815ab97197a41caab4e1da5e574a441ca38/src/patternfly/components/Dropdown/dropdown-toggle.hbs